edgelet/iotedge/src/config/import/mod.rs

```diff
@@ -215,19 +219,22 @@ fn execute_inner(old_config_file: &Path) -> Result<Vec<u8>, std::borrow::Cow<'st
                         },
                     ),
                 always_reprovision_on_startup,
```

Comment:
We had talked about differentiating between `true` and unset, so that unset maps to `false` rather than `true`.

ie you need to change the `old_config` type to be an `Option<bool>`

Reply:
Fixed